src/Core/Domain/Product/Stock/ValueObject/MovementReasonId.php

```diff
@@ -40,8 +40,13 @@ class MovementReasonId
      *
      * @todo: add other keys
      */
-    public const MOVEMENT_REASON_DECREASE_BY_EMPLOYEE_EDITION = 'PS_STOCK_MVT_INC_EMPLOYEE_EDITION';
-    public const MOVEMENT_REASON_INCREASE_BY_EMPLOYEE_EDITION = 'PS_STOCK_MVT_DEC_EMPLOYEE_EDITION';
+    public const INCREASE_BY_EMPLOYEE_EDITION_CONFIG_KEY = 'PS_STOCK_MVT_INC_EMPLOYEE_EDITION';
+    public const DECREASE_BY_EMPLOYEE_EDITION_CONFIG_KEY = 'PS_STOCK_MVT_DEC_EMPLOYEE_EDITION';
+
+    public const EMPLOYEE_EDITION_CONFIG_KEYS = [
+        self::INCREASE_BY_EMPLOYEE_EDITION_CONFIG_KEY,
+        self::DECREASE_BY_EMPLOYEE_EDITION_CONFIG_KEY,
+    ];
```

Comment:
Could employee edition reasons get ever extended ? If extension is required, a service method is a better solution than a simple constant.

Reply:
My answer would be: [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it).

This need does not exist yet, so we should not try to build a code that can handle a need that we are not sure will appear one day. AFAIK edition reasons have not changed for a long time so we should be safe with a simple constant, and this code will not be very hard to update if we need to.

Reply:
I wasn't clear, sorry. I meant customization, which is much easier by service injection. But I totally get your point o/